scripts/release-binary.sh

```diff
@@ -134,7 +134,7 @@ do
       fi
       ;;
     "debug")
-      CONFIG_PARAMS="--config=debug"
+      CONFIG_PARAMS=""
```

Comment:
Looks like something's not quite right; the binary doesn't exist where bazel expects. Try 
```suggestion
      CONFIG_PARAMS="-c dbg"
```

Reply:
thanks seems to work now!